.gitignore

```diff
@@ -43,3 +43,10 @@ local.properties
 
 ### VS Code ###
 .vscode/
+
```

Comment:
In my opinion, we need to add the files to ignore files that created while running the Gradle tasks.

Reply:
That files;
```suggestion
backend/build.txt
backend/install.txt
backend/test.txt
build.txt
frontend/build.txt
frontend/install.txt
frontend/test.txt
install.txt
test.txt
```

Reply:
Done.